docs/src/adoc/index.adoc

```diff
@@ -4094,6 +4094,13 @@ To use this plugin, add a Maven dependency:
 </dependency>
 ----
 
+Ensure the Kotlin compiler's https://kotlinlang.org/docs/reference/using-maven.html#attributes-specific-for-jvm[JVM target version] is set to 1.8:
```

Comment:
Does this advice apply even if you use a later jdk, say 11?

Reply:
I was seeing this problem on OpenJDK 10, the latest available in the Ubuntu repos. Installing 11 shows the same issue, though with a useful error message I wasn't seeing previously:
```
Cannot inline bytecode built with JVM target 1.8 into bytecode that is being built with JVM target 1.6. Please specify proper '-jvm-target' option
```

Reply:
How about let's say "at least 1.8" so we don't drift out of date.